cub/test/catch2_test_block_scan.cu

```diff
@@ -107,6 +107,24 @@ struct sum_op_t
   }
 };
 
+template <class T, scan_mode Mode>
+struct min_init_value_op_t
+{
+  T initial_value;
+  template <int ItemsPerThread, class BlockScanT>
+  __device__ void operator()(BlockScanT& scan, T (&thread_data)[ItemsPerThread]) const
```

Comment:
critical: the test doesn't seem to be covering value-based API. The following code doesn't compile at the moment:
```cpp
  int initial_value = 42;
  int thread_data = threadIdx.x;
  cub::BlockScan<int, 32>{}.InclusiveScan(thread_data, thread_data, initial_value, cub::Sum{});
```
leads to:
```
error: more than one instance of overloaded function "cub::WarpScan<T, LOGICAL_WARP_THREADS, LEGACY_PTX_ARCH>::InclusiveScan [with T=int, LOGICAL_WARP_THREADS=32]" matches the argument list
```
Could you please extend the test to cover this?
```cpp
  T thread_data/*[ItemsPerThread]*/;
```

Reply:
done